common/src/main/java/tachyon/Constants.java

```diff
@@ -279,6 +279,8 @@
       "tachyon.worker.tieredstore.levels";
   public static final String WORKER_FILESYSTEM_HEARTBEAT_INTERVAL_MS =
       "tachyon.worker.filesystem.heartbeat.interval.ms";
+  public static final String WORKER_FILE_PERSIST_POOL_SIZE =
+      "tachyon.worker.file.persiste.pool.size";
```

Comment:
persiste -> persist

also, this new config parameter should be added to the docs.